src/App.js

```diff
@@ -1,11 +1,16 @@
 //TODO: STEP 1 - Import the useState hook.
-import React from "react";
+import React, {useState} from "react";
 import "./App.css";
 import BottomRow from "./BottomRow";
 
 function App() {
   //TODO: STEP 2 - Establish your applictaion's state with some useState hooks.  You'll need one for the home score and another for the away score.
-
+  const [homeTd, scoreTd] = useState(0)
```

Comment:
Variable names make sense. Nice use of console.logs. Great job setting the initial state